recipes-bsp/atf/atf-96boards.inc

```diff
@@ -0,0 +1,26 @@
+inherit deploy
+
+DEPENDS = "openssl-native"
+
+# /usr/lib/atf/bl1.bin not shipped files. [installed-vs-shipped]
+INSANE_SKIP_${PN} += "installed-vs-shipped"
+
+export LDFLAGS=""
+
+do_compile() {
+    oe_runmake CROSS_COMPILE=${TARGET_PREFIX} all fip DEBUG=1 PLAT=${COMPATIBLE_MACHINE} SPD=none \
```

Comment:
can you rename the file to atf-common.inc or atf.inc? the file isn't 96boards specific and I don't want to give the impression it is 96boards only.